src/main/java/it/gov/pagopa/payment/instrument/service/idpaycode/EncryptIdpayCodeService.java

```diff
@@ -0,0 +1,13 @@
+package it.gov.pagopa.payment.instrument.service.idpaycode;
+
+import it.gov.pagopa.payment.instrument.dto.PinBlockDTO;
+
+public interface EncryptIdpayCodeService {
```

Comment:
Maybe this name could be better? this class is exposing many utility methods, not just one, thus I think that this name suggest an utility service
```suggestion
public interface IdpayCodeEncryptionService {
```